WordPress/src/test/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModelTest.kt

```diff
@@ -655,6 +664,43 @@ class WPMainActivityViewModelTest : BaseUnitTest() {
         assertThat(viewModel.mainActions.value!!.map { it.actionType }).isEqualTo(expectedOrder)
     }
 
+    @Test
+    fun `isMultipleSiteAvailable should be true when there are more than one site`() {
+        whenever(mSiteStore.sitesCount).thenReturn(2)
+        assertThat(viewModel.isMultipleSiteAvailable).isEqualTo(true)
+    }
+
+    @Test
+    fun `isMultipleSiteAvailable should be false when there is only one site`() {
+        whenever(mSiteStore.sitesCount).thenReturn(1)
+        assertThat(viewModel.isMultipleSiteAvailable).isEqualTo(false)
+    }
+
+    @Test
+    fun `isMultipleSiteAvailable should be false when there are no site`() {
+        whenever(mSiteStore.sitesCount).thenReturn(0)
+        assertThat(viewModel.isMultipleSiteAvailable).isEqualTo(false)
+    }
+
+    @Test
+    fun `firstSite should be return the first site available in the list of sites`() {
+        val sites = mock<ArrayList<SiteModel>>()
```

Comment:
@SowmiaS I think an import was missing in this file for the `mock` helper function, Please make sure to push it as well. 
import statement - `import com.nhaarman.mockitokotlin2.mock`